engibench/problems/thermoelastic3d/model/fem_plotting.py

```diff
@@ -0,0 +1,47 @@
+"""Module for plotting a 3D thermoelastic design with napari."""
+
+from __future__ import annotations
```

Comment:
```suggestion
```

Already requiring python >= 3.10. So this is no longer needed.
Also, this is in conflict with our constraint api which relies on the types of the signature.